src/util/utilization.jl

```diff
@@ -0,0 +1,126 @@
+
+"""
+    `run_UTIL(data::Dict{String,<:Any})`
+
+Returns a priority restoration order for damaged components in a single network.
+
+# Example
+```
+julia> run_UTIL(data)
+Dict{String, Any} with 6 entries:
+    "4" => [("branch", "2")]
+    "1" => [("gen", "1")]
+    "5" => [("branch", "3")]
+    "2" => [("gen", "2")]
+    "6" => [("gen", "3")]
+    "3" => [("branch", "1")]
+```
+"""
+function run_UTIL(data::Dict{String,<:Any})
+    if _IM.ismultinetwork(data)
+        Memento.error(_PM._LOGGER, "run_UTIL requires a single network.")
+    end
+
+    d_comp_vec = vcat([[(comp_type,comp_id) for comp_id in comp_ids] for (comp_type,comp_ids) in get_repairable_components(data)]...)
+    d_comp_cost = [_util_value(data,comp_type,comp_id) for (comp_type,comp_id) in d_comp_vec]
+    d_comp_vec = [d_comp_vec[i] for i in sortperm(d_comp_cost)] # reordered damaged component vector
```

Comment:
Do we need `sortperm` or would `sort` be sufficent?

Reply:
`sortperm` returns the indices of the cost vector, sorted by cost.  These indices are then used to select the component vector in the correct order.

`sort` simply returns a sorted list of the cost, but not the indices to reorder the component vector.

Reply:
I was thinking skipping `d_comp_cost` and doing something like this on `d_comp_vec` directly,
```
sort!(d_comp_vec, by=(x)->_util_value(data,x...))
```
If that has a syntax issue this should work,
```
sort!(d_comp_vec, by=(x)->_util_value(data,x[1],x[2]))
```

Reply:
That is a much cleaner syntax